src/main/java/frc/team2412/robot/commands/shooter/ShooterTargetCommand.java

```diff
@@ -22,7 +22,7 @@ public ShooterTargetCommand(ShooterSubsystem shooter, DoubleSupplier distanceSup
     @Override
     public void execute() {
         double distance = distanceSupplier.getAsDouble();
-        double yaw = yawSupplier.getAsDouble();
+        double yaw = yawSupplier.getAsDouble() + ShooterConstants.turretAngleBiasEntry.getDouble(0);
```

Comment:
Should verify that `turretAngleBiasEntry` is a reasonable value before applying it

Reply:
Since we're now using constants, I think we don't need to check.

Reply:
Also, since the method in ShooterSubsystem clamps the value anyway, I don't think this is really an issue (it also logs when the value is clamped)